### DISCUSSION TITLE
Enhancement: Please Integrate apipie.ai

### DISCUSSION THREAD

**Discussion by EncryptShawn** [danny-avila/LibreChat#2231] [Feature Requests & Suggestions]
### What features would you like to see added?

-The most affordable, reliable and fastest AI available
-One API to access ~500 Models and growing
-Language, embedding, voice, image, vision and more
-Global AI load balancing, route queries based on price or latency
-Redundancy for major models providing the greatest up time possible
-Global reporting of AI availability, pricing and performance

### More details

 Why bother integrating, together, openrouter, eden, etc? just integrate APIpie to access them all from one place and plus it also provides:

### Which components are impacted by your request?

General

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
If it's OpenAI api-spec compatible, then it's already "integrated"

if you want to add it to the list of known endpoints, submit a PR with a working config: https://docs.librechat.ai/install/configuration/ai_endpoints.html
